src/plugins/telemetryTable/TelemetryTable.js

```diff
@@ -115,71 +116,131 @@ define([
             this.filteredRows.sortBy(sortOptions);
         }
 
-        loadComposition() {
+        async loadComposition() {
             this.tableComposition = this.openmct.composition.get(this.domainObject);
+
             if (this.tableComposition !== undefined) {
-                this.tableComposition.load().then((composition) => {
+                let composition = await this.tableComposition.load();
```

Comment:
Why did this change?

Reply:
No clue, I'll change it back, it's not a necessary change.